js/src/components/social/FacebookView.js

```diff
@@ -0,0 +1,106 @@
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+import { Alert } from "@yoast/components";
+import { makeOutboundLink } from "@yoast/helpers";
+import { SocialMetadataPreviewForm } from "@yoast/social-metadata-forms";
+import { __, sprintf } from "@wordpress/i18n";
+import styled from "styled-components";
+
+const PremiumInfoText = styled( Alert )`
+	margin-top: 16px;
+ p {
+	 margin: 0;
+ }
+`;
+
+const YoastShortLink = makeOutboundLink();
+
+/* Translators: %s expands to the social medium name, Facebook.  */
+const previewText = sprintf(
+	__(
+		"Do you want to preview what it will look like if people share this post on %s? You can, with", "yoast-components"
```

Comment:
This is not a complete sentence to translate. I can see you add a bold "Yoast SEO Premium" somewhere, but translators can't.